thinkhazard/static/js/report.js

```diff
@@ -210,7 +97,40 @@
   function showOverview() {
     $('ul.hazards a').filter('[href="#overview"]').tab('show');
     hazardType = undefined;
-    addMap(hazardType);
+    showHazard();
     $('#legend').hide();
   }
+
+  /**
+   * Adds or replaces the hazard image layer.
+   */
+  function showHazard(hazardType) {
+    if (hazardLayer) {
+      map.removeLayer(hazardLayer);
```

Comment:
Can we use this basemap instead of stame watercolor?

https://a.tiles.mapbox.com/v4/ingenieroariel.m9a2h374/page.html?access_token=pk.eyJ1IjoiaW5nZW5pZXJvYXJpZWwiLCJhIjoibXhDZ3pIMCJ9.qTmPYCbnUKtaNFkvKKysAQ#3/0.00/0.00

On Mon, May 25, 2015 at 8:59 AM, Éric Lemoine <notifications@github.com>
wrote:

> In thinkhazard/static/js/report.js
> <https://github.com/GFDRR/thinkhazard/pull/136#discussion_r30981087>:
>
> >      $('#legend').hide();
> >    }
> > +
> > +  /**
> > +   * Adds or replaces the hazard image layer.
> > +   */
> > +  function showHazard(hazardType) {
> > +    if (hazardLayer) {
> > +      map.removeLayer(hazardLayer);
>
> There's a setSource method that we could use instead of re-creating the
> layer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GFDRR/thinkhazard/pull/136/files#r30981087>.
>